pymc3/distributions/multivariate.py

```diff
@@ -763,14 +763,20 @@ class LKJCorr(Continuous):
     """
 
     def __init__(self, eta=None, n=None, p=None, transform='interval', *args, **kwargs):
-        if p is not None:
+        if (p is not None) & (n is not None) & (eta is None):
+            warnings.warn('Reasigning input argument: shape parameter n -> eta; '
```

Comment:
"Reassigning". Also, I would start with: "Parameters to LKJCorr have, changed: shape parameter ... Please update your code. Automatically re-assigning parameters for backwards compatibility."